src/lib/pickles/compile.ml

```diff
@@ -376,6 +376,13 @@ struct
            name (Type_equal.Id.uid id) )
         log
 
+  type ('a, 'b, 'c) wrap_overrides =
+    { domain : Pickles_base.Proofs_verified.t option
+    ; main : ('a, 'b, 'c) wrap_main_generic option
+    }
+
+  let no_override = { domain = None; main = None }
```

Comment:
Would be good to comment about overrides, unless already present somewhere.

Reply:
I'm not too sure what they actually mean. I just refactored a bit of code to remove 1 argument from some function calls

Reply:
I have reverted the changes.